src/js/abstract/base-component.js

```diff
@@ -350,7 +351,11 @@ export default class BaseComponent extends HTMLElement {
           this._lightDOMRefs.forEach((ref) => {
             // Important: Once the light DOM is live it shouldn't be moved out
             // instead make sure to clone it for incremental updates
-            const refClone = ref.cloneNode(false);
+            const refClone = ref.cloneNode(true);
```

Comment:
@LucaMele 
Just this line seems to fix our issue, but by cloning including the whole subtree we will get some performance hits (to be honest I don't see them being observable).

Reply:
true.. and that wont work in IE